# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mp3info/hotfix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -urN mp3info-0.8.4-old/gmp3info.c mp3info-0.8.4-new/gmp3info.c
--- mp3info-0.8.4-old/gmp3info.c	2001-07-16 11:17:16.000000000 +0200
+++ mp3info-0.8.4-new/gmp3info.c	2003-09-08 05:22:37.000000000 +0200
@@ -79,8 +79,6 @@
 	        mp3.id3.genre[0]=gget_genre(tmp);
 	   	strcpy(tmp,gtk_entry_get_text(GTK_ENTRY(id3win_text_track)));
 		mp3.id3.track[0]=atoi(tmp);
-		if (mp3.id3.track[0] > 255)
-			mp3.id3.track[0]=255;
 	        write_tag(&mp3);
 	   }
 
@@ -484,12 +482,12 @@
 }
 
 void about_mp3info(void) {
-	quick_popup(VERSION,"MP3Info
-			An ID3 Tag Editor\n\n
-			Original author: Ricardo Cerqueira <rmc@plug.pt>
-			Current maintainer: Cedric Tefft <cedric@earthling.net>
-			GTK Version: Ricardo Cerqueira <rmc@plug.pt>\n\n
-			Ricardo Cerqueira, Cedric Tefft   1999-2001\n\n");
+	quick_popup(VERSION,"MP3Info\n"
+"			An ID3 Tag Editor\n\n\n"
+"			Original author: Ricardo Cerqueira <rmc@plug.pt>\n"
+"			Current maintainer: Cedric Tefft <cedric@earthling.net>\n"
+"			GTK Version: Ricardo Cerqueira <rmc@plug.pt>\n\n\n"
+"			Ricardo Cerqueira, Cedric Tefft   1999-2001\n\n");
 }
 
 /* rmcc has left the building */
